### PR TITLE
adds dracula+ and removes darcula

### PR DESCRIPTION
**Pull request by whym1here**
I first saw this theme on [windowsterminalthemes.dev](https://windowsterminalthemes.dev/?theme=Dracula%2B). 

**Font:** JetBrainsMono Nerd Font Mono
**Preview:**
dracula_plus

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ configuration file.
 |                                 **_Cobalt2_**<br>[source](https://github.com/wesbos/cobalt2/tree/master/Cobalt2)                                  |               ![Cobalt2](images/Cobalt2.png)               |
 |                                 **_cyber_punk_neon_**<br>[source](https://github.com/Roboron3042/Cyberpunk-Neon)                                  |       ![cyber_punk_neon](images/cyber_punk_neon.png)       |
 |                                                **_darcula_**<br>[source](https://draculatheme.com)                                                |               ![darcula](images/darcula.png)               |
+|                                                **_darcula_plus_**<br>[source](https://draculatheme.com)                                                |               ![darcula_plus](images/darcula_plus.png)               |
```
Comment (chrisduerr) [inline README.md:67]: It's dracula, not darcula, right?

Also would you mind deleting the other theme that made the same spelling mistake? We have a dracula theme in the repo and that doesn't seem much different.

**Review by chrisduerr — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (whym1here) [reply README.md]: Made the suggested changes. Do let me know if I need to make any more changes.

File: themes/dracula_plus.toml [hunk 2]
```diff
@@ -16,7 +16,7 @@ yellow = '#FFCB6B'
 blue = '#82AAFF'
 magenta = '#C792EA'
 cyan = '#8BE9FD'
-white = '#F8F9F2'
+white = '#BBBBBB'
```
Comment (chrisduerr) [inline themes/dracula_plus.toml:19]: Why did you change this?

**Review by chrisduerr — CHANGES_REQUESTED** [review]

File: themes/dracula_plus.toml (hunk 2 above)
Comment (whym1here) [reply themes/dracula_plus.toml:19]: The Light Gray was not visible in print_colors before this change.

Comment (chrisduerr) [reply themes/dracula_plus.toml:19]: The goal is to provide common themes though, regardless of their contrast or "quality". So changing them arbitrarily does not make sense.

Comment (whym1here) [reply themes/dracula_plus.toml:19]: Ok sure I will revert the change

**Review by chrisduerr — APPROVED** [review]
